### DISCUSSION TITLE
Error: Cannot find module '/hello-world/node_modules/strtok3/lib/index.js'

### DISCUSSION THREAD

**Discussion by tavaresgerson** [adonisjs/core#4686] [Help] [answered]
So, i started the update company's project and show this error:

```shel
[ info ] starting HTTP server...

   Error: Cannot find module '/home/tavares/hello-world/node_modules/strtok3/lib/index.js' imported from /home/tavares/hello-world/node_modules/file-type/index.js
 

   ⁃ finalizeResolution
     node_modules/ts-node/dist-raw/node-internal-modules-esm-resolve.js:366
   ⁃ moduleResolve
     node_modules/ts-node/dist-raw/node-internal-modules-esm-resolve.js:801
   ⁃ Object.defaultResolve
     node_modules/ts-node/dist-raw/node-internal-modules-esm-resolve.js:912
``` 

I only runned the command: `npm init adonisjs@latest hello-world --kit` e this error showed. :cry:

**Comment by Tonours** [comment, marked as answer]
Hello! I've had the same error and it seems that there is an issue with `strtok3` [issue #1143](https://github.com/Borewit/strtok3/issues/1143).

As a workaround until the `strtok3` author fixes the issue, you can specify the previous version of `strtok3` in your `package.json` by using [overrides](https://docs.npmjs.com/cli/v9/configuring-npm/package-json#overrides) or [resolutions](https://classic.yarnpkg.com/en/docs/selective-version-resolutions) depending on whether you're using NPM or Yarn.
 
**NPM:**
```json
"overrides": {
  "strtok3": "8.0.1"
}
```

**Yarn:**
```json
"resolutions": {
  "strtok3": "8.0.1"
}
```

**Comment by thetutlage** [comment]
Can you try creating a new project and verify that is works? We have pinned the strtok3 version to 8.0.1 since the author of the package has published incorrect file tree to npm

**Reply by tavaresgerson** [reply]
Yes, i create a new project and works, thank you very much! 🔥
